### ISSUE TITLE
Desyncs when generating quests and sometimes Error quest descriptions.

### ISSUE THREAD

**Issue by RaskaSlaanesh** [rwmt/Multiplayer#446]
We are playing a game of 4 people now.
We only have all the DLCs, Multiplayer and Harmony ( as its a dependency ). Some of us had modded games and simply just changed mods to this setup, so there might need to be file integration verification I suppose.

So to the issues.
After quests are generated, there might be quests that cause one of my friends to desync. Its only one friend, others are fine and none were disconnected by this. ( This is why i mentioned that its probably some issue with previous mods ), But either way, I included it, just in case something is found.

The other issue is on my own client.
Sometimes, a quest generates with an Error message as its name and description. Cant really determine which quest that actually is.
The error messages are some generic error messages, they dont have any stack trace in them.

Sometimes quest has a description, but errors ( seems like null references or something similar ) when referencing objects, locations, material, factions or pawns.

This is only in some, like 1 quest in 2 batches of geenerated quests what I noticed.

**Comment by Zankoas** [comment]
Yes getting the same here I think, example of the quest description error messages:
![image](https://github.com/user-attachments/assets/4a8cc54d-641f-4d67-86f2-37dc10b293f6)
![image](https://github.com/user-attachments/assets/25814854-bd59-42e7-a245-1ca48b06e9af)
![image](https://github.com/user-attachments/assets/f35d9fe3-d850-4a32-aa60-f0a1f0eb8526)
Shout if there is anything else useful I can provide

**Comment by samche2000** [comment]
Hello,

I have exactly same bugs on only some quests.

What do you need to help you to debug this ?

Thank you,
